apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,76 @@
-import { Button } from "~/libs/components/components.js";
+import logo from "~/assets/img/svg/sing-logo.svg";
```

Comment:
```suggestion
import logo from "~/assets/img/svg/auth-circle-logo.svg";
```
Or maybe some other name